### ISSUE TITLE
Unable to initialize bench in 5.13.0

### ISSUE THREAD

**Issue by dj12djdjs** [frappe/bench#1342]
```
 ~ bench init --verbose --skip-assets frappe-bench
Setting Up Environment
$ /home/ds2204/.local/bin/virtualenv  env -p python3
created virtual environment CPython3.10.4.final.0-64 in 160ms
  creator CPython3Posix(dest=/home/ds2204/frappe-bench/env, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/ds2204/.local/share/virtualenv)
    added seed packages: pip==22.1.2, setuptools==62.6.0, wheel==0.37.1
  activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
$ /home/ds2204/frappe-bench/env/bin/python -m pip install  --upgrade pip
Requirement already satisfied: pip in ./env/lib/python3.10/site-packages (22.1.2)
Collecting pip
  Using cached pip-22.2.1-py3-none-any.whl (2.0 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 22.1.2
    Uninstalling pip-22.1.2:
      Successfully uninstalled pip-22.1.2
Successfully installed pip-22.2.1
Traceback (most recent call last):
  File "/home/ds2204/.local/lib/python3.10/site-packages/bench/commands/make.py", line 68, in init
    init(
  File "/home/ds2204/.local/lib/python3.10/site-packages/bench/utils/render.py", line 105, in wrapper_fn
    return fn(*args, **kwargs)
  File "/home/ds2204/.local/lib/python3.10/site-packages/bench/utils/system.py", line 66, in init
    bench.setup.config(redis=not skip_redis_config_generation, procfile=not no_procfile)
  File "/home/ds2204/.local/lib/python3.10/site-packages/bench/utils/render.py", line 126, in wrapper_fn
    return fn(*args, **kwargs)
  File "/home/ds2204/.local/lib/python3.10/site-packages/bench/bench.py", line 376, in config
    generate_config(self.bench.name)
  File "/home/ds2204/.local/lib/python3.10/site-packages/bench/config/redis.py", line 21, in generate_config
    write_redis_config(
  File "/home/ds2204/.local/lib/python3.10/site-packages/bench/config/redis.py", line 66, in write_redis_config
    template = bench.config.env().get_template(template_name)
  File "/home/ds2204/.local/lib/python3.10/site-packages/bench/config/__init__.py", line 7, in env
    return Environment(loader=PackageLoader("bench.config"))
  File "/usr/lib/python3/dist-packages/jinja2/loaders.py", line 319, in __init__
    raise ValueError(
ValueError: The 'bench.config' package was not installed in a way that PackageLoader understands.

ERROR: There was a problem while creating frappe-bench
Do you want to rollback these changes? [y/N]:
```

**Comment by dj12djdjs** [comment]
Somehow, installing bench with git on the develop branch works. But not when it is installed via pip

**Comment by dj12djdjs** [comment]
Well, this is working in the recently released 5.13.1
